scripts/libraries/rpc.py

```diff
@@ -282,20 +282,20 @@ def loop(self, recv_timeout=1000, send_timeout=1000): # public
                 self.__schedule_cb = None
             if self.__loop_cb is not None: self.__loop_cb()
 
-def __get_can_settings(bit_rate, sampling_point):
+def __get_can_settings(bit_rate, sample_point):
```

Comment:
Is this needed anymore now that you put it inside the CAN init method?

Reply:
It can calculate bit timing from bitrate.